state-chain/runtime/src/runtime_apis.rs

```diff
@@ -125,7 +130,7 @@ decl_runtime_apis!(
 		fn cf_flip_supply() -> (u128, u128);
 		fn cf_accounts() -> Vec<(AccountId32, VanityName)>;
 		fn cf_account_flip_balance(account_id: &AccountId32) -> u128;
-		fn cf_account_info_v2(account_id: &AccountId32) -> RuntimeApiAccountInfoV2;
+		fn cf_validator_info(account_id: &AccountId32) -> RuntimeApiAccountInfoV2;
```

Comment:
it's a breaking change but it's a good time to do it with 1.3 coming soon and this old one being deprecated and only used for getting validator info